reference/constraints/Email.rst

```diff
@@ -135,7 +135,7 @@ address. Valid values are:
 
 * ``loose``, it uses a simple regular expression to validate the address (it
   checks that at least one ``@`` character is present, etc.). This validation is
-  too simple and it's recommended to use the ``html5`` validation instead;
+  very simple and it's recommended to use one of the other methods instead;
```

Comment:
I'm keeping "too" instead of "very" because I think it conveys a better meaning in this case.